### PR TITLE
Draft: bullets in footer

### PR DESCRIPTION
**Pull request by krlmlr**
``` r
palmerpenguins::penguins
#> # A tibble: 344 × 8
#>    species island    bill_length_mm bill_depth_mm flipper_le…¹ bod…² sex    year
#>    <fct>   <fct>              <dbl>         <dbl>        <int> <int> <fct> <int>
#>  1 Adelie  Torgersen           39.1          18.7          181  3750 male   2007
#>  2 Adelie  Torgersen           39.5          17.4          186  3800 fema…  2007
#>  3 Adelie  Torgersen           40.3          18            195  3250 fema…  2007
#>  4 Adelie  Torgersen           NA            NA             NA    NA <NA>   2007
#>  5 Adelie  Torgersen           36.7          19.3          193  3450 fema…  2007
#>  6 Adelie  Torgersen           39.3          20.6          190  3650 male   2007
#>  7 Adelie  Torgersen           38.9          17.8          181  3625 fema…  2007
#>  8 Adelie  Torgersen           39.2          19.6          195  4675 male   2007
#>  9 Adelie  Torgersen           34.1          18.1          193  3475 <NA>   2007
#> 10 Adelie  Torgersen           42            20.2          190  4250 <NA>   2007
#> • 334 more rows
#> • abbreviated variable names ¹​flipper_length_mm, ²​body_mass_g
palmerpenguins::penguins_raw
#> # A tibble: 344 × 17
#>    stud…¹ Sam…² Spe…³ Reg…⁴ Isl…⁵ Stage Ind…⁶ Clu…⁷ `Date Egg` Cul…⁸ Cul…⁹ Fli…˟
#>    <chr>  <dbl> <chr> <chr> <chr> <chr> <chr> <chr> <date>     <dbl> <dbl> <dbl>
#>  1 PAL07…     1 Adel… Anve… Torg… Adul… N1A1  Yes   2007-11-11  39.1  18.7   181
#>  2 PAL07…     2 Adel… Anve… Torg… Adul… N1A2  Yes   2007-11-11  39.5  17.4   186
#>  3 PAL07…     3 Adel… Anve… Torg… Adul… N2A1  Yes   2007-11-16  40.3  18     195
#>  4 PAL07…     4 Adel… Anve… Torg… Adul… N2A2  Yes   2007-11-16  NA    NA      NA
#>  5 PAL07…     5 Adel… Anve… Torg… Adul… N3A1  Yes   2007-11-16  36.7  19.3   193
#>  6 PAL07…     6 Adel… Anve… Torg… Adul… N3A2  Yes   2007-11-16  39.3  20.6   190
#>  7 PAL07…     7 Adel… Anve… Torg… Adul… N4A1  No    2007-11-15  38.9  17.8   181
#>  8 PAL07…     8 Adel… Anve… Torg… Adul… N4A2  No    2007-11-15  39.2  19.6   195
#>  9 PAL07…     9 Adel… Anve… Torg… Adul… N5A1  Yes   2007-11-09  34.1  18.1   193
#> 10 PAL07…    10 Adel… Anve… Torg… Adul… N5A2  Yes   2007-11-09  42    20.2   190
#> • 334 more rows
#> • 5 more variables
#> • abbreviated variable names ¹​studyName, ²​`Sample Number`, ³​Species, ⁴​Region,
#>   ⁵​Island, ⁶​`Individual ID`, ⁷​`Clutch Completion`, ⁸​`Culmen Length (mm)`,
#>   ⁹​`Culmen Depth (mm)`, ˟​`Flipper Length (mm)`
#> • variable names: `Body Mass (g)` <dbl>, Sex <chr>, `Delta 15 N (o/oo)` <dbl>,
#>   `Delta 13 C (o/oo)` <dbl>, Comments <chr>
```

<sup>Created on 2022-04-24 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

### CODE REVIEW

**Comment by hadley** [comment]
I know we're moving to bulleted lists elsewhere in messaging, but here I'm worried about it making the output too long. 

I'd prefer something like more like:

```R
#>  With 334 more rows, and 5 more variables: `Body Mass (g)` <dbl>, Sex <chr>, 
#>  `Delta 15 N (o/oo)` <dbl>, `Delta 13 C (o/oo)` <dbl>, Comments <chr>
#>  Abbreviated names  are:  ¹​studyName, ²​`Sample Number`, ³​Species, ⁴​Region,
#>   ⁵​Island, ⁶​`Individual ID`, ⁷​`Clutch Completion`, ⁸​`Culmen Length (mm)`,
#>   ⁹​`Culmen Depth (mm)`, ˟​`Flipper Length (mm)`
```

Do we ever truncate the abbreviated names? Or do we assume that there's unlikely to be that many of them?

**Comment by krlmlr** [comment]
Thanks. We truncate the abbreviated columns by limiting the number of footer lines. This also applies to the bullet list. Does that change your assessment?

If we keep the prose, are you suggesting to get rid of the leading `# ` hash?

**Comment by hadley** [comment]
We should keep the `#`; I think overall I just prefer the existing footer display so I'd prefer a more minimal set of changes to add the abbreviated column names.

File: tests/testthat/_snaps/tbl-format-footer.md [hunk 1]
```diff
@@ -26,10 +26,11 @@
         letters, collapse = " ") := 2)), width = 70))
     Output
       <tbl_format_footer(setup)>
-      # ... with 16 more variables, and variable names: l <chr>, m <chr>,
-      #   n <chr>, o <chr>, p <chr>, q <chr>, r <chr>, s <chr>, t <chr>,
-      #   u <chr>, v <chr>, w <chr>, x <chr>, y <chr>, z <chr>,
-      #   `a b c d e f g h i j k l m n o p q r s t u v w x y z` <dbl>
+      * 16 more variables
```
Comment (hadley) [inline tests/testthat/_snaps/tbl-format-footer.md:29]: Can you please restore the `#` and I'll re-review.

**Comment by hadley** [comment]
Fixes #582?

**Comment by krlmlr** [comment]
Superseded by #617.

**Pull request closed without merging**